### DISCUSSION TITLE
v1.15 Boot Issue

### DISCUSSION THREAD

**Discussion by mrbishi** [Joshua-Riek/ubuntu-rockchip#148] [General]
Thanks for the new build Joshua, getting the following on booting the new build though (both SD and NVMe):

![chrome_Tex6SGw8D0](https://github.com/Joshua-Riek/ubuntu-rockchip/assets/117778111/1cb632f5-022c-463e-a1a6-39220a4afe33)

Any ideas / does the onboard bootloader need an update etc to access the new build?

Cheers.

**Comment by Joshua-Riek** [comment]
OH, I know the issue, it's in the device tree, I forgot to fix it when I was working on the rock 5b. I will need to re-upload the release with the fix. I might be able to upload a single file to fix this, hold on.

**Reply by mrbishi** [reply]
Ah amazing, thanks for your help.

**Reply by Joshua-Riek** [reply]
For my reference, this kernel patch should fix the issue. I need to do a quick test to ensure it will not break other devices.

```diff
From 1d3a8aaa7f503c3ed66dda82afdb5dd5fffff094 Mon Sep 17 00:00:00 2001
From: Joshua Riek <jjriek@verizon.net>
Date: Fri, 19 May 2023 15:12:46 -0400
Subject: [PATCH] arm64: dts: remove chosen bootargs

---
 arch/arm64/boot/dts/rockchip/rk3588-linux.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3588-linux.dtsi
index d8c7c2465f77..97de58e8970f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3588-linux.dtsi
@@ -11,9 +11,9 @@ aliases {
 		mmc2 = &sdio;
 	};
 
-	chosen: chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xfeb50000 console=ttyFIQ0 irqchip.gicv3_pseudo_nmi=0 root=PARTUUID=614e0000-0000 rw rootwait";
-	};
+	//chosen: chosen {
+	//	bootargs = "earlycon=uart8250,mmio32,0xfeb50000 console=ttyFIQ0 irqchip.gicv3_pseudo_nmi=0 root=PARTUUID=614e0000-0000 rw rootwait";
+	//};
 
 	cspmu: cspmu@fd10c000 {
 		compatible = "rockchip,cspmu";
-- 
2.25.1

```

**Comment by Joshua-Riek** [comment]
I created an issue from this discussion post #149. I uploaded the kernel changes to my ppa, so building there might take 1-2 hours. Then another 3-4 hours for my release builds to finish. Thanks again for finding this.
